moveit_core/robot_model/include/moveit/robot_model/joint_model_group.h

```diff
@@ -581,6 +581,10 @@ class JointModelGroup
   bool isValidVelocityMove(const double* from_joint_pose, const double* to_joint_pose, std::size_t array_size,
                            double dt) const;
 
+  /** \brief Computes the indices of joint variables given a vector of joint names to look up */
+  bool computeJointVariableIndices(const std::vector<std::string>& joint_names,
+                                   std::vector<size_t>& joint_bijection) const;
+
```

Comment:
This seems to be an exact copy of `computeIKIndexBijection` - except throwing a different error message!
Increasing code redundancy should be strongly avoided!